### PR TITLE
feat(i18n): update zh-CN texts

### PR DESCRIPTION
**Pull request by breakeryu**
update libraries and serial-moitor texts.

### CODE REVIEW

**Comment by breakeryu** [comment]
@Does anyone is here?

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/guides/libraries.md [hunk 1]
```diff
@@ -1,16 +1,17 @@
...
 
-For example, the following file will install the latest versions of Servo and FastLED, as well as version 2.3.0 of MySensors:
+在默认情况下，Wokwi使用标准内置Arduino库（如Wire.h和SPI.h）编译你的代码。
+
+要将第三方库添加到项目中，请将“libraries.txt”文件添加到您的项目中。列出您想要包含的库，每行一个库。开头为“#”的行是注释。
+
+例如，以下文件将安装最新版本的 Servo 库和 FastLED 库，以及 MySensors 库的 2.3.0 版本：
 
 ```
 # Sample libraries.txt file:
```
Comment (urish) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/guides/libraries.md:17]: Can you please translate this title too?

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/guides/libraries.md [hunk 2]
```diff
@@ -21,6 +22,19 @@ FastLED
 MySensors@2.3.0
 ```
 
-You can find the library names in the Arduino Library Manager. Alternatively, you can find it inside the source code of the library. Look for the `name` field of the `library.properties` file.
+You can find the library names in the Arduino Library Manager. Alternatively, you can find it inside the source code of the library. Look for the `name` field of the `library.properties` file.（这句话不太懂什么意思？？）
```
Comment (urish) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/guides/libraries.md:25]: You wrote some question in Chinese, I guess it is not part of the translation. How can I help?

**Review by urish — CHANGES_REQUESTED** [review]
Thanks for the translation!

I see that the source files you translated are a bit old, they have been updated in the English version since they were copies from there.

Do you need help with finding the changes?

Also, I left two comments on the translation itself.

**Comment by breakeryu** [comment]
thankyou，urish。I will check again。

**Comment by breakeryu** [comment]
hi,urish. How can I solve this requested changes probelm? 
I have checked my docs and added a new pull requeset.

**Pull request closed without merging**